.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "trailingComma": "none"
```

Comment:
May you propose stylistic changes in another PR?

Reply:
I removed the "trailingComma": "none". And added commas, where the prettier wants it.

Reply:
Could you explain why you want to remove the prettierrc config file and change the code style?

Reply:
I removed it as it is "Stylistic changes"

Other things were changed automatically by "npm run format"